Cargo.toml

```diff
@@ -15,30 +14,19 @@ exclude = [
 [lib]
 crate-type = ["cdylib", "rlib"]
 
-[profile.release]
-opt-level = 3
-debug = false
-rpath = false
-lto = true
-debug-assertions = false
-codegen-units = 1
-panic = 'abort'
-incremental = false
-overflow-checks = true
-
 [features]
 # for more explicit tests, cargo test --features=backtraces
 backtraces = ["cosmwasm-std/backtraces"]
 # use library feature to disable all instantiate/execute/query exports
 library = []
 
 [dependencies]
-cosmwasm-std = { version = "0.16.0" }
-cosmwasm-storage = { version = "0.16.0" }
-cw-storage-plus = "0.8.0"
+cosmwasm-std = { version = "0.16.2" }
+cw2 = { path = "../../packages/cw2", version = "0.8.1" }
+cw-storage-plus = { path = "../../packages/storage-plus", version = "0.8.1", features = ["iterator"] }
```

Comment:
features = ["iterator"] is the default now, and can be omitted

Reply:
Done.